hoomd/mesh.py

```diff
@@ -75,6 +103,15 @@ def _remove_dependent(self, obj):
             if self._added:
                 self._remove()
 
+    @property
+    def name(self):
+        """(`str`): Mesh name."""
+        return self.types[0]
+
+    @name.setter
+    def name(self, name):
+        self.types = [name]
+
```

Comment:
Do you expect to extend to multiple types in version 3? If so this will introduce problems.

Reply:
For now, yes. I might go for multiple triangle types later, though.

Reply:
Then we should remove the name argument, and assign a default name in `types`. That way, when a future change adds support for multiple types, there is a clearly defined default behavior that the user can override.

Reply:
done